core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -465,6 +470,10 @@ export class WorkflowActionService {
   public unhighlightLinks(...links: string[]): void {
     this.getJointGraphWrapper().unhighlightLinks(...links);
   }
+  public highlightCommentBoxes(multiSelect: boolean, ...commentBoxIDs: string[]): void {
```

Comment:
The formatting is off. Between different methods there need to be a blank space.